docs/service-catalog/docs/040-service-binding-usage-custom-resource.md

```diff
@@ -0,0 +1,60 @@
+
+```
+kubectl get crd servicebindingusages.servicecatalog.kyma.cx -o yaml
+```
+
+## Sample custom resource
+
+This is a sample resource in which the ServiceBindingUsage injects a Secret associated with the `redis-instance-binding` ServiceBinding to the `redis-client` Deployment in the `production` Namespace. This example has the **conditions.status** field set to `true`, which means that the ServiceBinding injection is successful. If this field is set to `false`, the **message** and **reason** fields appear.
+
+```
+apiVersion: servicecatalog.kyma.cx/v1alpha1
+kind: ServiceBindingUsage
+metadata:
+ name: redis-client-binding-usage
+ namespace: production
+spec:
+ serviceBindingRef:
+   name: redis-instance-binding
+ usedBy:
+   kind: deployment
+   name: redis-client
+ parameters:
+   envPrefix:
+     name: "pico-bello"
+status:
+    conditions:
+    - lastTransitionTime: 2018-06-26T10:52:05Z
+      lastUpdateTime: 2018-06-26T10:52:05Z
+      status: "True"
+      type: Ready
+```
+
+## Custom resource parameters
+
+This table lists all the possible parameters of a given resource together with their descriptions:
+
+
+| Parameter   |      Mandatory?      |  Description |
+|:----------:|:-------------:|:------|
+| **metadata.name** |    **YES**   | Specifies the name of the CR. |
+| **metadata.namespace** |    **YES**   | Specifies the Namespace in which the CR is created. |
+| **spec.serviceBindingRef.name** |    **YES**   | Specifies the name of the ServiceBinding. |
+| **spec.usedBy** |    **YES**   | Specifies the application into which the Secret is injected. |
+| **spec.usedBy.kind** |    **YES**   | Specifies the name of the [UsageKind](041-usage-kind-custom-resource.md). |
+| **spec.usedBy.name** |    **YES**   | Specifies the name of the application. |
+| **spec.parameters.envPrefix** |    **NO**   | Defines the prefix of environment variables injected by the ServiceBindingUsage. The prefixing is disabled by default. |
+| **spec.parameters.envPrefix.name** |    **YES**   | Specifies the name of the prefix. This field is mandatory if the **envPrefix** is specified.  |
+| **status.conditions** |    **NO**   | Specifies the state of the ServiceBindingUsage.|
+| **status.conditions.lastTransitionTime** |    **NO**   | Specifies the time when the Binding Usage Controller processes the ServiceBindingUsage for the first time or when the status field changes. |
+| **status.conditions.lastUpdateTime** |    **NO**   | Specifies the time of the last ServiceBindingUsage update. |
+| **status.conditions.status** |    **NO**   |  Specifies if the ServiceBinding injection is successful. |
+| **status.conditions.type** |    **NO**   | Defines whether the condition is `ready` or not. |
```

Comment:
it's more like this one: `Defines the type of the condition. Currently, only `Read` type is supported`